openedx/core/lib/xblock_utils/__init__.py

```diff
@@ -108,7 +108,7 @@ def wrap_xblock(
         )
     ]
 
-    if isinstance(block, (XModule, XModuleDescriptor)):
+    if isinstance(block, (XModule, XModuleDescriptor)) or getattr(block, 'uses_xmodule_styles_setup', False):
```

Comment:
Using the same setup means we do not have to touch any styles or make any adjustments.